### ISSUE TITLE
start_card not functioning with version 4.0

### ISSUE THREAD

**Issue by manonstreet** [bramkragten/swipe-card#22]
Version 3.2.1 honors the start_card parameter, but after upgrading to version 4, the swiper always starts on card 1.  I've downgraded back to 3.2.1 and confirm the feature is now functional again.

**Comment by manonstreet** [comment]
Interestingly, the editor preview does show the referenced start code even though it is not shown in my lovelace dashboard.

**Comment by bramkragten** [comment]
Yeah, it is a timing thing probably, will have a quick look soon.

**Comment by manonstreet** [comment]
Bump.  Any chance you can take a look?

**Comment by BunpGhost** [comment]
For me, not only start_card but a strange glitch where changing views would force to the last card.
In version 3.2.1, everything works as expected!

**Comment by manonstreet** [comment]
You are right - downgrading to 3.2.1 does fix both issues for me as well

**Comment by IanAdd** [comment]
Still broken - Start_card

**Comment by Jbautista13** [comment]
> Still broken - Start_card

EDIT 3: Webkit browsers seems to have an issue where if you have cssMode set to true in the swiper-card parameters configuration the `await this._cardPromises;` isn't enough. Instead of `this._cardPromises` I tried `await getCardSize()` which includes two `await` calls inside, one for `await this._cardPromises;` which we were using previously, and another which waits to get the height of all the cards. I haven't found any scenarios so far where start_card doesn't work with `await getCardSize()` added.

EDIT 2: I've redone the test by recreating the same simple swiper scenario and this time start_card is working perfectly. I'll keep testing.

EDIT: I've done some more testing on different swiper scenarios and it seems that the below suggestion isn't fullproof in all cases. This seems to work in scenarios where you have cards that take a while to load, but if you have simple cards while it will start on a card other than the first, it seems to be random which card it starts on, maybe there is some other condition we should be waiting for before swiper is initialized? Cards loading into the DOM possibly?

I've been messing around with the code found in the [src/index.js](https://github.com/bramkragten/swipe-card/tree/fcd46ed34463524a07fd2d7e783fbae699c0389b/src) file and it seems adding `await this._cardPromises;` inside of the `async initialLoad()` function before the Swiper Instance is created `this.swiper = new Swiper(...)` solves some of the issues.

I have no idea what exactly adding `await this._cardPromises;` does, it might just slightly delay the code execution by an arbitrary amount which fixes the timing issue Bram was talking about, but in my testing it seems pretty bulletproof other bugs notwithstanding.

My thinking behind adding the code `await this._cardPromises` was that it seemed as though without it we were telling Swiper to go to element X in the Slide before they had time to be created.

I'm not well versed enough in JS to understand if adding `await this._cardPromises` actually makes the code wait until all the cards are loaded before initializing Swiper, that was simply my thought process.
